boskos/cmd/cli/cli.go

```diff
@@ -140,6 +150,42 @@ Examples:
 	acquire.Flags().DurationVar(&options.acquire.timeout, "timeout", 0*time.Second, "If set, retry this long until the resource has been acquired")
 	root.AddCommand(acquire)
 
+	release := &cobra.Command{
+		Use:   "release",
+		Short: "Release resource leases",
+		Long: `Release a resource lease, blocking.
+
+Resources should have their leases released when they are finished
+with being used. Identify which resource lease to release by name
+and determine what state the resource should be in when the lease
+is released.
+
+Examples:
+
+  # Release a lease on "my-thing" and mark it dirty when releasing
+  $ boskosctl release --name my-thing --target-state dirty`,
+		Run: func(cmd *cobra.Command, args []string) {
+			options.initializeClient()
+			err := options.c.ReleaseOne(options.release.name, options.release.targetState)
```

Comment:
@krzyzacy @sebastienvas so right now the "client library" is very opinionated on how it's used. This does not make a call to release anything because there's nothing in the in-memory store. You guys OK with exporting `client.release()` and just using that?

Reply:
sure

Reply:
OK, second commit here does that